glightbox.module

```diff
@@ -56,6 +56,17 @@ function glightbox_menu() {
   return $items;
 }
 
+/**
+ * Implements hook_init().
+ */
+function glightbox_init() {
+  // Create selector(class) that is unique for the current document.
+  // This is done for security reasons so that only the links generated
+  // by this module are processed by the javascript plugin. Manually 
+  // injected links are thus ignored.
+  config_set('glightbox.settings', 'selector', 'glightbox-' . backdrop_random_key(8));
```

Comment:
This seems expensive and would slow down every request.
I think it would make more sense to generate this in theme_glightbox() and not store it in config.

Reply:
Yea, I see. I've made a new commit where I moved the selector generation to the theme registering function.